code/modules/mob/living/simple_animal/slime/slime.dm

```diff
@@ -552,5 +552,10 @@
 		LAZYREMOVE(GLOB.mob_spawners[spawner], src)
 	GLOB.poi_list -= src
 
+/mob/living/simple_animal/slime/proc/make_master(mob/user)
+	Friends[user]++
```

Comment:
You'll probably need a higher number, like 20. For example, `SLIME_FRIENDSHIP_ATTACK` is 8.